h/h_api/bulk_api/model/data_body.py

```diff
@@ -73,7 +77,11 @@ def create(cls, user_id, group_ref):
     def member_id(self):
         """The user which is a member of this group."""
 
-        return self.relationships["member"]["data"]["id"]
+        _member_id = self._member_id
```

Comment:
Why the local variable?

Reply:
Yeah, this whole bit is kind of goofy. I think we might want an id ref object that unifies all four of these accessors so it's `member.id`, `member.ref` instead of `member_ref`.

Reply:
I've changed this now so there's a single object which is called twice, making the whole thing much simpler.